interfaces/portal/src/app/pages/project-registrations/project-registrations.page.ts

```diff
@@ -128,7 +135,14 @@ export class ProjectRegistrationsPageComponent {
             }),
           ),
         );
+
         window.open(getOriginUrl() + url, '_blank');
+
+        this.trackingService.trackEvent({
+          category: TrackingCategory.manageRegistrations,
+          action: TrackingAction.selectContextMenuOption,
+          name: `open-in-new-tab`,
```

Comment:
pff it took me some time to figure out what/where this action was. I had no idea, and the name 'contextMenu' did nog help in any way ;) Anyway, this is pre-existing, so no comment for this PR.